### PR TITLE
feat: add managers category (yum, apt, homebrew etc.)

### PR DESCRIPTION
**Pull request by tabrindle**
`node src/cli.js --managers --system`

```
  System:
    OS: macOS 10.14.5
    CPU: (8) x64 Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz
    Memory: 433.11 MB / 16.00 GB
    Shell: 5.3 - /bin/zsh
  Managers:
    Cargo: 1.31.0 - ~/.cargo/bin/cargo
    CocoaPods: 1.7.3 - /usr/local/bin/pod
    Composer: 1.8.6 - /usr/local/bin/composer
    Gradle: 5.5 - /usr/local/bin/gradle
    Homebrew: 2.1.7 - /usr/local/bin/brew
    Maven: 3.6.1 - /usr/local/bin/mvn
    pip: 19.0.3 - /usr/local/bin/pip
    RubyGems: 2.5.2.3 - /usr/bin/gem
```

### CODE REVIEW

**Comment by tabrindle** [comment]
@gengjiawen Take a look if you get a chance.

File: src/helpers/managers.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+  getMavenInfo: () => {
+    utils.log('trace', 'getMavenInfo');
+    return Promise.all([
+      utils.run('mvn --version').then(utils.findVersion),
+      utils.which('mvn').then(utils.condensePath),
+    ]).then(v => utils.determineFound('Maven', v[0], v[1]));
+  },
+
+  getpipInfo: () => {
```
Comment (gengjiawen) [inline src/helpers/managers.js]: Can we handle pip3 here like Python3 on macOS and linux.

Comment (tabrindle) [reply src/helpers/managers.js]: Yeah! Was wondering if that was overdoing it or not.

**Comment by gengjiawen** [comment]
Hey, any plan to make a new release ?

**Pull request merged**